xml/article_virtualization.xml

```diff
@@ -24,20 +24,20 @@
 
    <para>
     To see the degree of interference from running within a particular &kvm;
-    configuration versus running on bare-metal configuration, Each RT
+    configuration versus running on bare-metal configuration, each RT
```

Comment:
```suggestion
    configuration versus running on a bare-metal configuration, each RT
```

Reply:
Good catch. "Been missed" is ungrammatical, too; replaced with "being missed".